mantis-runtime/src/main/java/io/mantisrx/runtime/sink/ServerSentEventRequestHandler.java

```diff
@@ -65,6 +66,7 @@
     private static final String SAMPLE_PARAM = "sample";
     private static final String SAMPLE_PARAM_MSEC = "sampleMSec";
     private static final String CLIENT_ID_PARAM = "clientId";
+    private static final String SOURCE_JOB_NAME_PARAM = "sourceJobName";
```

Comment:
naming nit: `sourceJobName` -> `targetJobCluster` as the SSE sink could be on any job

Reply:
The first commit actually uses `targetJob` but we had a bit of a debate in a Slack thread because Raven currently uses `sourceJobName`.

Reply:
No strong feeling either way. Just pointing out `sourceJobName` is what's used in raven's `TargetInfo` json.